components/alert.css

```diff
@@ -0,0 +1,82 @@
+@import url(/main.css);
+@import url(/components/badges.css);
+
+.alerts {
+  padding: 1rem;
+  margin: 0.5rem;
+  background-color: white;
+  border-radius: 5px;
+}
+
+h6 {
+  padding: 1rem 0.5rem;
+}
+.alert-primary {
+  display: flex;
+  width: 600px;
+  padding: 0.5rem;
+  margin: 10px 0;
+  background-color: #c2c2c2;
```

Comment:
you can use CSS variable it will be convenient for your usage and also recommended